modules/custom/adobe_analytics/adobe_analytics.libraries.yml

```diff
@@ -0,0 +1,14 @@
+analytics-prod:
+  header: true
+  js:
+    https://assets.adobedtm.com/cAdobeAnalyticscec67651710193d2331efef325107c23a0145/satelliteLib-92fcbbc24eeebb654a3dfbe688ed2dfe10a53e24.js: { type: external }

```

Comment:
please add attributes such as preprocess, minified etc. For example: https://github.com/open-data/gcweb_bootstrap/blob/master/gcweb.libraries.yml